scripts/mutect2_wdl/unsupported/README.md

```diff
@@ -22,8 +22,8 @@ The following resources:
 * A reference .fasta file, along with accompanying .fasta.fai and .dict files.
 * A gatk4 java .jar file.
 * A Picard java .jar file.
-* Three lists of .bam files -- one each for 5-plex, 10-plex and 20-plex replicates -- where each row has the format <bam_file.bam></TAB><bam_index.bai></TAB><sample_name>
-* A list of .bam files of the specificity validation's replicates.
+* Three lists of .bam files -- one each for 5-plex, 10-plex and 20-plex replicates -- where each row has the format <bam_file.bam></TAB><bam_index.bai>
```

Comment:
Out of curiosity, could this cross pairing be done in the WDL with a `cross` call?

Reply:
Seems to me the answer is yes, once wdl gets a composite bam + index data type, but for now it would probably be contrived.